### PR TITLE
004 style: added nunjucks template to add Kainos styling to pages

### PR DESCRIPTION
**Pull request by erinmcj**
### Describe your changes
Added nunchucks template to add Kainos styling to all pages. Html pages can now inherit the nunjucks template. 

## Issue ticket number and link
US026
https://tasks.office.com/kainos.com/en-US/Home/Planner/#/plantaskboard?groupId=3a4655dc-9106-4ae5-b900-b6ebe83dfab5&planId=Ww6vdHsxYkuv4tMevZj1_ZYAFksm

#### General
- [x] I have performed a self-review of my code.
- [x] Code changes are considered complete for the given story (i.e., the comments from code review are addressed).
- [x] The commit title follows conventional commit standards.
- [x] I have put the PR link into the planner.
- [x] The PR has a clear and descriptive title and a detailed description of the changes.
- [x] Code builds cleanly without any errors or warnings.
 
#### Code Quality
- [x] I have run linting and there were no errors.
 
#### Testing
- [x] New functionality is tested.
- [x] All unit and integration tests passed.
- [x] I have run accessibility tests and there were no errors.
 
#### Deployment
- [x] Documentation has been updated where needed.
 
#### Security
- [x] No sensitive data is hard-coded or exposed.

### CODE REVIEW

**Review by thomkainos — DISMISSED** [review]

File: src/app.ts [hunk 1]
```diff
@@ -17,6 +17,8 @@ app.use(bodyParser.urlencoded({
   extended: true
 }))
 
+app.use(express.static('views'));
```
Comment (swalker125) [inline src/app.ts]: so I would take a look at those express docs again - it's using a directory "public". I would follow that pattern so move all these images out of views :) 

and then we defo want something like this app.use(express.static(path.join(__dirname, 'public'))) so we can always find the folder.

Note in the docs -> it has /static -> that defines a url where the images can be found. If you go localhost:3000/images/Flux_White_Footer.png right now it'll load. I don't hate putting it all under static. So maybe just copy whats in those docs

File: views/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```
Comment (swalker125) [inline views/index.html]: so looks like nunjucks is set up -> https://mozilla.github.io/nunjucks/templating.html

take a look at that link. It talks about template inheritance and how we can reuse templates

you should be able to create some sort of file aka base.njk

which says something like (excuse my sudo code)

```
<header stuff>

<main>
{ % block content %} {% endblock % }
</main>

<footer stuff>
```
and then use this like

```
{% extends "base.njk" %}


{% block content %}

<the stuff that thomas is working on / any future pages>

{% endblock %}
```

Now you have a reusable template which we can put on all the pages :)

File: src/app.ts [hunk 3]
```diff
@@ -29,4 +31,8 @@ app.listen(3000, () => {
     console.log('Server started on port 3000');
 });
 
-app.get('/', getAllDatabases);
\ No newline at end of file
+app.get('/', getAllDatabases);
+
+app.get('/index', async (req: express.Request, res: express.Response) => {
+    res.render('index.html');
```
Comment (swalker125) [inline src/app.ts:38]: do you need to specify .html here?

**Review by swalker125 — COMMENTED** [review]
Few improvements, but ultimately, good work!

File: src/app.ts (hunk 3 above)
Comment (swalker125) [reply src/app.ts:38]: nunjucks docs has this - so i'll close comment :)

File: src/app.ts (hunk 1 above)
Comment (swalker125) [reply src/app.ts]: this caused so much more drama than I thought it would... will resolve @erinmcj

**Review by thomkainos — APPROVED** [review]

**Review by swalker125 — APPROVED** [review]

**Pull request merged**